<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measurements</class>
 <widget class="QWidget" name="Measurements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurements</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_64">
       <item>
        <widget class="QLabel" name="label_62">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>CenterlineIds Array:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCenterlineIdsArray">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_62">
       <item>
        <widget class="QLabel" name="label_60">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Radius Array:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRadiusArray">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_67">
       <item>
        <widget class="QLabel" name="label_63">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Reconstructed CenterlineIds Array:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelReconCenterlineIdsArray">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_68">
       <item>
        <widget class="QLabel" name="label_61">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Reconstructed Radius Array:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelReconRadiusArray">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_38">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Normal to Normal</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Lesion Length (mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLesionLengthNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Lesion Lumenal Volume (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLesionLumenalVolumeNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Reconstructed Lumenal Volume (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelReconLumenalVolumeNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lumenal Volume Delta (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLumenalVolDeltaNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Volume Ratio (%):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVolumeRatioNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Volume Ratio per unit Length(% mm^-1):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVolumeRatioPerUnitLengthNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_80">
          <item>
           <widget class="QLabel" name="label_80">
            <property name="text">
             <string>Adjusted Volume Ratio (% mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAdjustedVolumeRatioNTN">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>FWHM</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_72">
          <item>
           <widget class="QLabel" name="label_72">
            <property name="text">
             <string>Lesion Length (mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLesionLengthFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_73">
          <item>
           <widget class="QLabel" name="label_73">
            <property name="text">
             <string>Lesion Lumenal Volume (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLesionLumenalVolumeFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_74">
          <item>
           <widget class="QLabel" name="label_74">
            <property name="text">
             <string>Reconstructed Lumenal Volume (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelReconLumenalVolumeFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_75">
          <item>
           <widget class="QLabel" name="label_75">
            <property name="text">
             <string>Lumenal Volume Delta (mm^3):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLumenalVolDeltaFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_76">
          <item>
           <widget class="QLabel" name="label_76">
            <property name="text">
             <string>Volume Ratio (%):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVolumeRatioFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_77">
          <item>
           <widget class="QLabel" name="label_77">
            <property name="text">
             <string>Volume Ratio per unit Length(% mm^-1):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVolumeRatioPerUnitLengthFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_79">
          <item>
           <widget class="QLabel" name="label_79">
            <property name="text">
             <string>Adjusted Volume Ratio (% mm):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAdjustedVolumeRatioFWHM">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>N/A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Morphological</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_37">
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>Stenosis Radius (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelStenosisRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_61">
        <item>
         <widget class="QLabel" name="label_59">
          <property name="text">
           <string>Proximal Normal Radius (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelProximalNormalRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Distal Normal Radius (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDistalNormalRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_41">
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>Degree of Stenosis (NASCET) (%):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDoSNascet">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_66">
        <item>
         <widget class="QLabel" name="label_66">
          <property name="text">
           <string>Distance Factor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDistanceFactor">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_33">
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Hausdorff Distance (Recon to Lesion) (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHausdorffDistanceReconToLesion">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_34">
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>Hausdorff Distance (Lesion to Recon) (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelHausdorffDistanceLesionToRecon">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_35">
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>RMS Distance (Recon to Lesion) (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labellabelRmsDistanceReconToLesion">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_36">
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>RMS Distance (Lesion to Recon) (mm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRmsDistanceLesionToRecon">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_65">
     <item>
      <widget class="QLabel" name="label_64">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Point Data Array:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_65">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Point Data Array</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_57">
        <item>
         <widget class="QLabel" name="label_55">
          <property name="text">
           <string>Maximum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPointDataMaximum">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_58">
        <item>
         <widget class="QLabel" name="label_56">
          <property name="text">
           <string>Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPointDataMinimum">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_59">
        <item>
         <widget class="QLabel" name="label_57">
          <property name="text">
           <string>Gradient Maximum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPointDataGradientMaximum">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_60">
        <item>
         <widget class="QLabel" name="label_58">
          <property name="text">
           <string>Gradient Minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPointDataGradientMinimum">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_63">
     <item>
      <widget class="QPushButton" name="pushButtonUpdate">
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
